src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -2535,8 +2663,8 @@ void KaleidoScope_Update(GlobalContext* globalCtx) {
 
             gSegments[8] = VIRTUAL_TO_PHYSICAL(pauseCtx->iconItemSegment);
 
-            for (i = 0; i < ARRAY_COUNTU(gItemAgeReqs); i++) {
-                if ((gItemAgeReqs[i] != 9) && (gItemAgeReqs[i] != ((void)0, gSaveContext.linkAge))) {
+            for (i = 0; i < ARRAY_COUNTU(gItemIconAgeInfo); i++) {
+                if ((gItemIconAgeInfo[i] != 9) && (gItemIconAgeInfo[i] != ((void)0, gSaveContext.linkAge))) {
```

Comment:
```suggestion
                if ((gItemIconAgeInfo[i] != AGE_REQ_NONE) && (gItemIconAgeInfo[i] != ((void)0, gSaveContext.linkAge))) {
```

or even
```suggestion
                if (!((gItemIconAgeInfo[i] == AGE_REQ_NONE) || (gItemIconAgeInfo[i] == ((void)0, gSaveContext.linkAge)))) {
```
to make it look more like the other similar conditionals